pkg/export/export.go

```diff
@@ -379,20 +402,24 @@ func New(ctx context.Context, logger log.Logger, reg prometheus.Registerer, opts
 		lease = NopLease()
```

Comment:
question: Why do we need the `Default()` call here?

Isn't all of this being done as part of the flags setup already?

In general, if clients of the struct are expected to do some operations on it before using it, I think it makes sense to wrap those operations in a constructor, i.e. `newExporterOpts`.